core/Loc.cc

```diff
@@ -49,31 +49,38 @@ Loc::Detail Loc::pos2Detail(const File &file, uint32_t off) {
         ENFORCE_NO_TIMER(false);
     }
     auto lineBreaks = file.lineBreaks();
+    // lower_bound is what the C++ STL calls the "least upper bound," i.e., a pointer to the first
+    // element which is greater than or equal to you in a sorted list
     auto it = absl::c_lower_bound(lineBreaks, off);
     if (it == lineBreaks.begin()) {
         detail.line = 1;
         detail.column = off + 1;
         return detail;
     }
+    detail.line = (it - lineBreaks.begin()) + 1;
```

Comment:
What do you think about using `std::distance` here instead?
```suggestion
    detail.line = std::distance(lineBreaks.begin(), it) + 1;
```